torchci/pages/api/clickhouse/[queryName].ts

```diff
@@ -0,0 +1,13 @@
+import { queryClickhouseSaved } from "lib/clickhouse";
+import type { NextApiRequest, NextApiResponse } from "next";
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  const queryName = req.query.queryName as string;
+
+  const response = await queryClickhouseSaved(queryName, req.query);
```

Comment:
Is there a mismatch here where you await `queryClickhouseSaved` here but the function is not marked as async?

Reply:
It returns a promise.  AFAIK you need the await keyword if you await things inside of you, but queryClickhouseSaved doesn't await inside, it just passes on the promise.  Not sure if that is good practice tho

Reply:
I see. It works passing the promise around.  I usually keep async/await in matching pair, i.e.

```
async queryClickhouseSaved {
    return await queryClickhouse()
}

...

const response = await queryClickhouseSaved
```

Maybe it's just a matter of preference.

Reply:
Changed `queryClickhouseSaved` to be async and to await the promise inside.  

I assumed it would be better to pass the promise around but idk anything about promises and this does seem to make it more explicit